BrowserCache_Environment_Nginx.php

```diff
@@ -131,12 +127,14 @@ public function generate( $mime_types, $cdnftp = false ) {
 
 			$impoloded = implode( '|', $exceptions );
 			if ( !empty( $impoloded ) ) {
-				$wp_uri = network_home_url( '', 'relative' );
-				$wp_uri = rtrim( $wp_uri, '/' );
+				$file_code = rtrim( network_home_url( '', 'relative' ), '/' ) . '/index.php?$args;';
+
+				if( $this->c->get_integer( 'browsercache.no404wp.response.exceptions' ) ){
+					$file_code = "=" . $this->c->get_integer( 'browsercache.no404wp.response.exceptions' );
```

Comment:
Use single quotes for strings that do not need character translation or escaping other single quotes -- `'='`.